packages/apollo-voyager-server/package.json

```diff
@@ -24,6 +24,7 @@
   },
   "dependencies": {
     "@aerogear/apollo-voyager-context": "^1.0.0",
+    "@aerogear/apollo-voyager-metrics": "^1.0.0",
```

Comment:
I don't see any changes in the apollo-voyager-server package. Is there any reason why we need this here?